.github/actions/upload-build-artifacts/action.yml

```diff
@@ -1,14 +1,25 @@
 name: Upload Build Artifacts
 description: Upload artifacts for each platform
+inputs:
+  gh_token:
+    required: true
 
 runs:
   using: 'composite'
   steps:
     - uses: actions/upload-artifact@v3
       with:
-        name: Mac Artifact
+        name: Mac Artifact (${{ runner.arch }})
         path: dist/apps/zui/*.dmg
 
+    - name: Merge latest-mac.yml Mac release files for Intel/ARM64
```

Comment:
Would go with either `Intel/Apple` or `amd64/arm64`.

Reply:
@mattnibs: As long as I'm changing it, how would you feel about `x86/arm64`? I ask because that matches what `electron-builder` shows for the `arch` value in respective builds. E.g., Intel:

```
  • packaging       platform=darwin arch=x64 electron=30.0.1 appOutDir=/Users/runner/work/zui/zui/dist/apps/zui/mac
```

Apple Silicon:

```
  • packaging       platform=darwin arch=arm64 electron=30.0.1 appOutDir=/Users/runner/work/zui/zui/dist/apps/zui/mac-arm64
```